Sources/SwiftCompilerPluginMessageHandling/PluginMessageCompatibility.swift

```diff
@@ -11,8 +11,8 @@
 //===----------------------------------------------------------------------===//
 
 /// Old compiler might send '.declaration' as "freeStandingDeclaration".
-@_spi(PluginMessage) public extension PluginMessage.MacroRole {
-  init(from decoder: Decoder) throws {
+@_spi(PluginMessage) extension PluginMessage.MacroRole {
```

Comment:
Shouldn't the `@_spi` attribute be moved as well?

Reply:
Yes, could you file an issue for that at https://github.com/apple/swift-format?

Reply:
https://github.com/apple/swift-format/issues/714